docs/develop/worker-performance.mdx

```diff
@@ -36,10 +37,175 @@ The Workflow Cache is created and shared between all the workers. It's designed
 
-## Task Queues Processing Tuning
+## Monitor Task Queue backlog metrics {#task-queue-metrics}
+
+A [Task Queue](https://docs.temporal.io/workers#task-queue) is a lightweight, dynamically allocated queue.
+[Worker Entities](https://docs.temporal.io/workers#worker-entity) poll the queue for [Tasks](https://docs.temporal.io/workers#task).
+The Temporal Service dynamically creates different [Task Queue types](/workers#task-queue) including Activity Task Queues, Workflow Task Queues, and Nexus Task Queues.
+These Task Queue types route their Tasks to Workers for Task completion.
+
+With an accurate estimate of backlog Tasks, you can determine the optimal number of Workers to deploy.
+Balance your Worker count with the number of Tasks to achieve the best performance.
+This approach minimizes Task backlog saturation and reduces idle Workers.
+
+Task Queue metrics provide numerical insights into your Task Queue activity and backlog characteristics.
+Use these metrics to tune your production deployments.
+Evaluate your Worker loads and assess whether you need to scale up or reduce your Worker deployment.
+
+### Task Queue types {#task-queue-types}
+
+For each Task Queue name, Temporal creates separate queues for each Task Queue type, namely:
+
+- **Activity Task Queue**: A queue that holds Activity Tasks.
+  Activity Tasks represent units of work within a larger business process.
+  Each Activity Task contains the context required by an Activity Definition.
+  Workers poll Activity Tasks from the Activity Task Queue and use them to initiate Activity Executions.
+- **Workflow Task Queue**: A queue that holds Workflow Tasks.
+  Workflow Tasks contain the context needed by a Workflow Definition.
+  Workers poll Workflow Tasks from the Workflow Task Queue and use them to initiate Workflow Executions.
+- **Nexus Task Queue** ([Public Preview](/evaluate/development-production-features/release-stages#public-preview)): A queue that holds Nexus Tasks.
+  Nexus Tasks are used for units of work passed between Namespaces.
+  Workers configured with the Nexus Service poll the Nexus Task Queue for available tasks and handle them by initiating Workflow Executions in the target Namespace.
+  Each Nexus Task contains the context required by the target Workflow Definition.
+
+Each Task Queue type provides unaggregated metrics.
+
+### Task Queue metrics {#task-queue-metrics-list}
+
+The Temporal Service reports information separately for each Task Queue type (not aggregated).
+Use the following metrics to retrieve detailed information about Task Queue health and performance.
+Available metrics include:
+
+- [`ApproximateBacklogCount`](#ApproximateBacklogCount)
+- [`ApproximateBacklogAge`](#ApproximateBacklogCount)
```

Comment:
```suggestion
- [`ApproximateBacklogAge`](#ApproximateBacklogAge)
```

Reply:
Fixing. Thank you!!!! What a great catch